src/main/java/com/sngular/kloadgen/extractor/extractors/JsonExtractor.java

```diff
@@ -204,31 +225,37 @@ private List<FieldValueMapping> extractMapInternalFields(
           FieldValueMapping.builder()
                            .fieldName(name)
                            .fieldType(value.getType() + "-map" + ((StringUtils.isNotEmpty(breadCrumb) && breadCrumb.endsWith("[:]")) ? "-map" :
-                               (StringUtils.isNotEmpty(breadCrumb) && breadCrumb.endsWith("[]")) ? "-array" : breadCrumb))
+                                                                                                                                                  (StringUtils.isNotEmpty(
```

Comment:
Check if the formatter its ok, the concatenation of this one should be in one or two lines instead of this